build.sh

```diff
@@ -43,7 +43,7 @@ LAST_COMMIT_TIMESTAMP=$(git log -1 --format=%ct)
 major=1
 minor=0
 # no. of days since epoch
-build=$(($LAST_COMMIT_TIMESTAMP/3600/24))
+build=0
```

Comment:
Making this change will require us to delete all of the deb packages currently in our feed, or nobody will be able to download any of the new packages.

I think if we bumped the minor number to 1, we would not need to do that. 
Thoughts?